pkgs/by-name/go/goto/package.nix

```
@@ -0,0 +1,65 @@
+{
+  coreutils,
+  fetchFromGitHub,
+  gawk,
+  goto,
+  lib,
+  nix-update-script,
+  runCommand,
+  stdenvNoCC,
+}:
+
+let
+  owner = "iridakos";
+  rev = "b7fda54e0817b9cb47e22a78bd00b4571011cf58";
+
+in
+stdenvNoCC.mkDerivation {
```

Let's use `finalAttrs` to improve override flexibility, and retrieve bound `tag` with derivation `version`
```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```